content/en/docs/tasks/observability/metrics/querying-metrics/test.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154,SC2155
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+
+# @setup profile=demo
+kubectl label namespace default istio-injection=enabled --overwrite
+
+# Install Bookinfo sample
+startup_bookinfo_sample
+
+# Install Prometheus
+kubectl apply -f samples/addons/prometheus.yaml -n istio-system
+_wait_for_deployment istio-system prometheus
```

Comment:
Remember that the main goal of this test is to confirm that the commands in the doc are working, so you should
try to test as many generated snips as possible.

```suggestion
_wait_for_deployment istio-system prometheus

_very_like snip_querying_istio_metrics_1 "$snip_querying_istio_metrics_1_out"
```

Reply:
Ok. Besides that I'll add some prometheus related verification as well.

Reply:
Extra verification is probably overkill, since the following snips will fail if things aren't right. The main point is that this test should be trying to test that what's documented is correct (i.e., run this command and you should see this output). If the doc is outdated and the command or output snips are a little different from what you actually get when you run them, we are trying to detect that kind of error. A side effect is we get extra testing of the istio functionality itself, but that's not the main goal for these tests.

Reply:
Oh. Yeah.  Sorry. I meant the prometheus related snippet in the same task.Added it.